README.md

```diff
@@ -42,9 +42,9 @@ if not functions -q fisher
 end
 ```
 
-### Changing the installation prefix
+### Package files location
```

Comment:
@Odaeus I prefer the original title! 🙏

Reply:
I appreciate that and I addressed it in my [second comment at the top of this page](https://github.com/jorgebucaran/fisher/pull/541#issuecomment-513539568). It negates the purpose of my effort to document this problem if a reader will most likely skip over it because the heading sounds like an esoteric action that they would have no interest in performing.

Please close this PR if you do not wish to change the heading as I don't see another way 
to make it prominent enough to help new users avoid data loss.

Reply:
I'll add your changes anyway, but I am open meet you in the middle with a different title. I agree that "Changing the installation prefix" sounds "advanced". What about "Changing the installation path" or "Customizing the installation path". They don't fully address your original intent, but sound a bit less intimidating.

Reply:
"Prefix" does indeed make it sound advanced and you're right that "path" is a more natural choice, especially as it matches the variable name. Though I still believe many people would skip over the section with a verb like "change" or "customize" in the belief that they would have no reason to change the default. Whereas a heading with a noun phrase appears more informational.
As one last attempt, I suggest the verbs "setting" or "managing" as both sound more equivocal on whether it's something that should be changed or not.